ddtrace/opentelemetry/span.go

```diff
@@ -7,16 +7,14 @@ package opentelemetry
 
 import (
 	"encoding/binary"
-	"encoding/json"
 	"errors"
-	"strconv"
-	"strings"
-	"sync"
-
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+	"strconv"
```

Comment:
This is not properly formatted. The standard library packages should be grouped together.